TTPChallenge/DataAnalystVC.swift

```diff
@@ -0,0 +1,63 @@
+//
+//  DataAnalystVC.swift
+//  TTPChallenge
+//
+//  Created by Mirim An on 2/15/17.
+//  Copyright © 2017 TeamMDC. All rights reserved.
+//
+
+import UIKit
+
+class DataAnalystVC: UIViewController {
+
+    @IBOutlet var informationButtons:[UIButton]!
+    
+    @IBOutlet weak var retakeQuizButton: UIButton!
+    override func viewDidLoad() {
+        super.viewDidLoad()
+
+        for button in informationButtons {
+            button.layer.cornerRadius = 12
+        }
+        retakeQuizButton.layer.cornerRadius = 20
+
+        // Do any additional setup after loading the view.
+    }
+
+    override func didReceiveMemoryWarning() {
+        super.didReceiveMemoryWarning()
+        // Dispose of any resources that can be recreated.
+    }
+    
+    @IBAction func informationBtnTapped(_ sender: UIButton) {
+        
+        let index = sender.tag
+        switch index {
+        case 0:
+            print("open safari for 1")
```

Comment:
here is how you open a safari view controller

https://developer.apple.com/reference/safariservices/sfsafariviewcontroller